pkg/pods/manifest.go

```diff
@@ -102,6 +103,21 @@ func ManifestFromBytes(bytes []byte) (*Manifest, error) {
 	return manifest, nil
 }
 
+func ManifestFromURI(manifestUri string) (*Manifest, error) {
+	// Fetch manifest (could be URI) into temp file
+	localMan, err := ioutil.TempFile("", "tempmanifest")
```

Comment:
I never noticed this before.

I wonder how many `tempmanifest`-s my laptop has by now...